BlackJackets-ui/src/App.jsx

```diff
@@ -61,6 +65,8 @@ function App() {
                 />
                 <Route path="/edit-venue/:id" element={<VenueEdit />} />
                 <Route path="/add-venue" element={<VenueAdd />} />
+                <Route
+                 <Route path="/message-list" element={<Notification />} />
```

Comment:
There is an extra `<Route` here that should be removed.

Reply:
Done